package.json

```diff
@@ -38,6 +38,12 @@
 		"javaExtensions": [
 			"./jars/com.redhat.quarkus.jdt.core.jar"
 		],
+		"languages": [{
+			"id": "quarkus-properties",
+			"extensions": [".properties"],
```

Comment:
nope, that's still too broad. If you open any .properties file, it's considered a quarkus properties file

Reply:
I think the spring boot extension does this as well.
![image](https://user-images.githubusercontent.com/20326645/63278627-9fd4db00-c275-11e9-9248-942062190d89.png)

I created an empty `application.properties` file in an empty folder and the spring boot extension loaded up:
![image](https://user-images.githubusercontent.com/20326645/63278689-c266f400-c275-11e9-8a99-35ce2cab70c3.png)

Reply:
try with foo.properties

Reply:
the spring boot extension doesn't start up with that name. VSCode only sees it as a Properties file. Is it possible to determine if we are in a Quarkus project without starting the extension and getting information from jdt.ls?

Reply:
It's complicated. We'd prolly need to issue a command to jdt.ls to determine if we're in a quarkus project, so we'd need to start vscode-quarkus. catch-22